### PR TITLE
Add stability check for registry track

### PR DESCRIPTION
**Pull request by jennyliang220**

### CODE REVIEW

**Review by deniak — CHANGES_REQUESTED** [review]
stability.js is not checking the additional sentence needed in CRY and CRYD.

**Comment by jennyliang220** [comment]
> stability.js is not checking the additional sentence needed in CRY and CRYD.

The implementation of this feature will be in a new PR, after https://github.com/w3c/tr-design/pull/271 is addressed.

File: lib/rules/sotd/draft-stability.js [hunk 1]
```diff
@@ -8,7 +8,11 @@ const self = {
 exports.name = self.name;
 
 exports.check = function (sr, done) {
-    if (!sr.config.stabilityWarning || sr.config.stabilityWarning === 'REC')
+    if (
```
Comment (deniak) [inline lib/rules/sotd/draft-stability.js:11]: I don't know why this hasn't been removed earlier but if a check doesn't do anything for specific profiles, then the rule ought to be removed from the profile itself exactly like how you are doing it for CRY.

File: lib/profiles/TR/CRY.js [hunk 2]
```diff
@@ -1,14 +1,17 @@
 exports.name = 'CRY';
 exports.config = {
     status: 'CRY',
-    longStatus: 'Candidate Registry Snapshot',
+    longStatus: 'Candidate Registry',
```
Comment (deniak) [inline lib/profiles/TR/CRY.js:4]: I don't understand how the tests passed with the previous longStatus. Are we even testing this?

**Review by deniak — CHANGES_REQUESTED** [review]

**Review by deniak — APPROVED** [review]

**Pull request merged**